.ci/install_runtime.sh

```diff
@@ -77,7 +78,17 @@ if [ "$USE_VSOCK" == "yes" ]; then
 	fi
 fi
 
-echo "Add runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to updates."
-docker_options="-D --add-runtime kata-runtime=/usr/local/bin/kata-runtime"
-echo "Add kata-runtime as a new/default Docker runtime."
-"${cidir}/../cmd/container-manager/manage_ctr_mgr.sh" docker configure -r kata-runtime -f
+if [ "$KATA_HYPERVISOR" == "qemu" ]; then
```

Comment:
@GabyCT looks like we made it either or. Can we support both at the same time, potentially one of them via a shell script. I will work on eliminating the shell script based on 
https://github.com/kata-containers/runtime/issues/1082

Till then can we route fc through a shell script